### PR TITLE
bpo-35970: Add help flag to base64 util

### PR DESCRIPTION
**Pull request by rkuska**
Probably a really unnecessary contribution but I failed today to print help message for base64 utility without an error:

```
 $ python3 -m base64 -help
option -h not recognized
usage: /usr/local/Cellar/python/3.7.2_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/base64.py [-d|-e|-u|-t] [file|-]
        -d, -u: decode
        -e: encode (default)
        -t: encode and decode string 'Aladdin:open sesame
```
So I felt like this is a ripple in time space continuum worth adjusting. I understand that this part of code is probably a long forgotten and therefore I will not be surprised if you decide to just close this PR.

https://bugs.python.org/issue35970

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by remilapeyre** [comment]
Hi @rkuska, thanks for your contribution!

This is bug and all bugs are worth fixing!

There is some process to follow before submitting changes to Python, you can read about them at https://devguide.python.org/pullrequest/#quick-guide

You first need to create an account on https://bugs.python.org/ and open a new issue.

Your Pull Request will also need a test before being accepted.

You should also use https://blurb-it.herokuapp.com/ to add a news entry to your PR.

**Comment by rkuska** [comment]
I've created an issue and changed commit message and also edited PR title and lastly news entry was added. I will add also tests once I will get some ack about my fix - not sure if getopt shouldn't be replaced by argparse.

File: Lib/base64.py [hunk 1]
```diff
@@ -558,22 +558,25 @@ def decodestring(s):
 def main():
     """Small main program"""
     import sys, getopt
+    usage = """usage: %s [-h|-d|-e|-u|-t] [file|-]
```
Comment (remilapeyre) [inline Lib/base64.py]: I think you can convert this to an f-string

**Review by remilapeyre — COMMENTED** [review]
I'm not sure getopt needs to be replaced, optparse has been deprecated but not getopt.

File: Misc/NEWS.d/next/Library/2019-02-11-19-06-10.bpo-35970.ZRvh51.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Add help flag to base64 cmd utility
```
Comment (remilapeyre) [inline Misc/NEWS.d/next/Library/2019-02-11-19-06-10.bpo-35970.ZRvh51.rst]: Add `Patch contributed by ...`

**Review by remilapeyre — CHANGES_REQUESTED** [review]

**Review by taleinat — CHANGES_REQUESTED** [review]
This certainly would be a nice improvement.

Please add two simple tests checking that the usage text is printed:
1. Running with `-h`
2. Running with an invalid parameter

(See `TestMain` in `test_base64.py`)

**Comment by csabella** [comment]
@rkuska, please address the code review for @taleinat.  Thank you!

**Comment by taleinat** [comment]
Ping, @rkuska?

**Comment by ammaraskar** [comment]
Oops, I think I accidentally closed this while trying to update rkuska's branch. Let me update this work in a new PR.

**Comment by ammaraskar** [comment]
@taleinat would you mind giving https://github.com/python/cpython/pull/28774, a continuation of this PR a quick review?

**Pull request closed without merging**